src/components/AQIDial.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import GaugeChart from 'react-gauge-chart';
+import {Text, Box, Tag, Grid, GridItem, Link} from '@chakra-ui/react';
+
+/**
+ * Interface for the props of the dial
+ * - currentReading is the aqi value that the dial should display
+ */
+interface DialProps {
+  currentReading: string;
+}
+
+/**
+ * AQI Dial Display Component
+ */
+const AQIDial: ({currentReading}: DialProps) => JSX.Element = ({
+  currentReading,
+}: DialProps) => {
+  /**
+   * The dial takes a percentage (in range 0 to 1). This function
+   * converts our AQI readings to the appropriate number between 0 and 1 that
+   * the dial should display
+   * @param currentReading - the aqi value to be displayed
+   */
+  const convertToPercent = (currentReading: string): number => {
+    // The maximum aqi value for the dial (corresponds to Health Alert for all)
+    const aqiMax = 250;
+    const dialMax = 1;
+    const aqi: number = +currentReading; // Convert from string to number
+    let dialReading = 0; // Initialize dial reading
+    if (aqi <= aqiMax) {
+      dialReading = aqi / aqiMax; // Convert to something between 0 and 1
+    } else {
+      // AQI is above health alert
+      dialReading = dialMax; // Set dial to 1 (max)
+    }
+    return dialReading;
+  };
+
+  return (
+    <Box>
+      <GaugeChart
```

Comment:
Is it possible to make the dial handle reach the edge more, like this but obviously prettier
<img width="184" alt="aqi-dial-extension" src="https://user-images.githubusercontent.com/31805145/107421246-91fcc080-6ace-11eb-965a-c064d6c1522c.PNG">

Reply:
I can't bring the needle to the arc but I can bring the arc to the needle: 
<img width="536" alt="image" src="https://user-images.githubusercontent.com/44306107/107433108-c2982680-6add-11eb-87d5-61527d966b8e.png">

Or something in between:
<img width="480" alt="image" src="https://user-images.githubusercontent.com/44306107/107433170-db084100-6add-11eb-8478-b67021e2355e.png">

Thoughts?